src/components/Todo/Todo.module.css

```diff
@@ -1,4 +1,13 @@
 .title {
   font-family: 'Courier New', Courier, monospace;
   text-align: center;
+  font-weight: bold;
+  font-size: 35px;
 }
+
+
+@media (max-width: 780px) {
+  .title {
+   font-size: 20px;
```

Comment:
Codacy found an issue: [Expected indentation of 4 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/ReactProject_karolina/pullRequest?prid=6313319)